app/models/user.rb

```diff
@@ -73,6 +73,13 @@ module Permissions
   scope :ungreeted, -> { where(greeted: false) }
 
   ####################
+  # Class method(s)  #
+  ####################
+  def self.search(term)
```

Comment:
This method name doesn't communicate the specific kind of search (email). I suggest either naming it `search_by_email`, or adding a second parameter — `field` or similar` that has a default value of `:email`, and then write the query so that it works with any field.

Being able to also search by `real_name` would be a very useful thing to add (not necessarily in this PR).